### PR TITLE
Hotfix/Tabs and Select style bug

### PR DESCRIPTION
**Pull request by hujiahao-hjh**
## What type of PR is this?

- [ ] Feature
- [x] Bugfix
- [ ] Test
- [ ] Documentation
- [ ] Refactor
- [ ] Style
- [ ] Chore

## Which issue(s) this PR fixes:

Fixes # fix Tabs and Select style bug

## Does this PR introduce a user interface change?

- [ ] Yes(screenshot is required)
- [x] No

### CODE REVIEW

File: shell/app/styles/antd-extension.scss [hunk 1]
```diff
@@ -407,6 +407,14 @@ div.ant-tabs-nav .ant-tabs-tab {
   margin-right: 40px;
 }
 
+div.ant-tabs-nav::before, .ant-tabs-nav::after {
+  display: block;
+}
+
+.task-drawer-main .task-drawer-tabs .ant-tabs-nav {
+  padding: 0 20px;
```
Comment (daskyrk) [inline shell/app/styles/antd-extension.scss:415]: This is not a common style, move to component related scss file.

File: shell/app/styles/antd-extension.scss [hunk 2]
```diff
@@ -407,6 +407,14 @@ div.ant-tabs-nav .ant-tabs-tab {
   margin-right: 40px;
 }
 
+div.ant-tabs-nav::before, .ant-tabs-nav::after {
+  display: block;
```
Comment (daskyrk) [inline shell/app/styles/antd-extension.scss:411]: What's this for?

Comment (hujiahao-hjh) [reply shell/app/styles/antd-extension.scss:411]: antd3.x styles makes display table, causing the the border not to work.

File: shell/app/modules/project/common/components/issue/edit-issue-drawer.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.issue-drawer-tabs .ant-tabs-nav {
+    padding: 0 20px;
```
Comment (daskyrk) [inline shell/app/modules/project/common/components/issue/edit-issue-drawer.scss:2]: Incorrect indent. style-lint not work?

**Review by laojun — APPROVED** [review]

File: shell/app/modules/project/common/components/issue/edit-issue-drawer.scss (hunk 3 above)
Comment (hujiahao-hjh) [reply shell/app/modules/project/common/components/issue/edit-issue-drawer.scss:2]: There's something wrong with the style-lint and it doesn't work.

**Pull request merged**